src/pages/LoginPage.js

```diff
@@ -12,10 +12,18 @@ class LoginPage extends Component {
       email: '',
       nickname: '',
       buttonDisable: true,
+      score: 0,
     };
     this.verifyInputs = this.verifyInputs.bind(this);
     this.handleClick = this.handleClick.bind(this);
     this.handleChange = this.handleChange.bind(this);
+    this.startLocalStorage = this.startLocalStorage.bind(this);
+  }
+
+  startLocalStorage() {
+    const { score } = this.state;
+    const obj = { player: { score } };
+    return localStorage.setItem('state', JSON.stringify(obj));
```

Comment:
Eae. se a função vai setar algo no localStorage, ela não precisa retornar nada, é tipo uma função com this.setState, não precisamos retornar a chamada dessa função, pois ela só está servindo para alterar algo lá no localStorage, e não pra retornar um valor.
```suggestion
    ocalStorage.setItem('state', JSON.stringify(obj));
```